WalletWasabi.Gui/Config.cs

```diff
@@ -84,6 +86,24 @@ public class Config : ConfigBase
 		[JsonConverter(typeof(EndPointJsonConverter), Constants.DefaultRegTestBitcoinP2pPort)]
 		public EndPoint RegTestBitcoinP2pEndPoint { get; internal set; } = new IPEndPoint(IPAddress.Loopback, Constants.DefaultRegTestBitcoinP2pPort);
 
+		[DefaultValue(false)]
+		[JsonProperty(PropertyName = "JsonRpcServerEnabled", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public bool JsonRpcServerEnabled { get; internal set; }
+
+		[DefaultValue("")]
+		[JsonProperty(PropertyName = "JsonRpcUser", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public string JsonRpcUser { get; internal set; }
+
+		[DefaultValue("")]
+		[JsonProperty(PropertyName = "JsonRpcPassword", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public string JsonRpcPassword { get; internal set; }
+
+		[JsonProperty(PropertyName = "JsonRpcServerPrefixes")]
```

Comment:
Shouldn't it be removed? Does it make sense to have hosts other than localhost? We're not encrypting the communication or anything like that.

Reply:
There is no need to restrict things so much imo. What if I want to publish this in `http://api.mymachine.org:41111/`? Why not?